scripts/vitest-config/package.json

```diff
@@ -13,11 +13,11 @@
   "dependencies": {
     "@modern-js/tsconfig": "workspace:*",
     "@types/node": "^14",
-    "@types/lodash": "^4.14.195",
-    "execa": "^5.1.1",
+    "@types/lodash": "4.14.181",
```

Comment:
How about upgrade all to latest version and unlock it.

Reply:
"@types/lodash": "^4.14.202"  is this ok?

Reply:
done